packages/tools/pie-icons-webc/build.js

```diff
@@ -14,78 +14,51 @@ const { icons } = pieIcons.default;
 
 const componentTemplate = (name, svg) => {
     const svgClasses = svg.match(/class="(.+?)"/)?.[1];
-
     const isLargeIcon = name.endsWith('Large');
     const sizeType = isLargeIcon ? 'LargeIconSize' : 'RegularIconSize';
+    // 32 or 'xs'
+    const defaultSize = isLargeIcon ? largeIconSizeDefault : `'${regularIconSizeDefault}'`;
```

Comment:
Would this not work?
```suggestion
    const defaultSize = isLargeIcon ? largeIconSizeDefault : regularIconSizeDefault;
```

Reply:
<img width="481" alt="Screenshot 2024-04-26 at 17 22 25" src="https://github.com/justeattakeaway/pie/assets/16766185/14d1592e-a4b4-4cb2-8c30-685e31986bce">
Sadly it seems to not compile to a string when I try this approach 😢